ovos_bus_client/session.py

```diff
@@ -299,6 +303,20 @@ def __init__(self, session_id: str = None, expiration_seconds: int = None,
         ]
         self.context = context or IntentContextManager()
 
+        if not stt_prefs:
+            stt = Configuration().get("stt", {})
+            sttm = stt.get("module", "ovos-stt-plugin-server")
+            stt_prefs = {"plugin_id": sttm,
+                         "config": stt.get(sttm) or {}}
+        self.stt_preferences = stt_prefs
+
+        if not tts_prefs:
+            tts = Configuration().get("tts", {})
+            ttsm = tts.get("module", "ovos-tts-plugin-server")
+            tts_prefs = {"plugin_id": ttsm,
+                         "config": tts.get(ttsm) or {}}
+        self.tts_preferences = tts_prefs
```

Comment:
Are these specs documented somewhere?

Reply:
its a standard mycroft.conf

Reply:
Huh, didn't realize you could override!

Reply:
Probably because you're just now adding it 😁